src/lib/xhr-helpers.js

```diff
@@ -0,0 +1,41 @@
+export const read = async url => {
```

Comment:
You can have something like a config file with the server url. Each of these calls can prefix the url provided. The final result should look something like: `fetch(`${serverUrl}/url`, ...)`